source/troubleshooting.txt

```diff
@@ -173,3 +173,141 @@ HOMEDRIVE, HOMEPATH, or USERPROFILE are blank
 
 This error might appear if the {+atlas-cli+} can't access your home
 directory.
+
+.. _local-deploy-errors:
+
+Troubleshoot Local Atlas Deployment Issues
+------------------------------------------
```

Comment:
Can we please add this new item as well?
https://docs.google.com/document/d/1XebNKyWXZ1t9xcizIEnb4oFtBp2a5C_RlpbY8fi8FCQ/edit#bookmark=id.frwbj7yi4gqs
cc @boooczek

Reply:
@blva, this is already covered. https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-32120/troubleshooting/#error--exit-status-127

Reply:
Hi! I'm talking about this one: "Atlas deployments commands are stuck"

Reply:
Thanks I didn't notice, I'll add one small suggestion